tempto-examples/bin/run_on_docker.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -ex
+
+# http://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-osx
+function absolutepath() {
+    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
+
+function retry() {
+  END=$(($(date +%s) + 600))
+
+  while (( $(date +%s) < $END )); do
+    set +e
+    "$@"
+    EXIT_CODE=$?
+    set -e
+
+    if [[ ${EXIT_CODE} == 0 ]]; then
+      break
+    fi
+    sleep 5
+  done
+
+  return ${EXIT_CODE}
+}
+
+function hadoop_master_container(){
+  docker ps --format '{{.Names}}' | grep hadoop-master
+}
+
+function check_hive() {
+  docker exec $(hadoop_master_container) hive -e 'show tables'
+}
+
+function docker_compose_network() {
+  docker network ls | grep docker | cut  -f 1 -d ' '
+}
+
+function check_presto() {
+  docker run \
+    --rm -it \
+    --net $(docker_compose_network) \
+    teradatalabs/presto-cli \
+    java -jar /presto-cli.jar --server presto-master:8080 \
+    --execute "SHOW CATALOGS"  | grep -i hive
+}
+
+function run_product_tests() {
+  docker run \
+    --rm -it \
+    --net $(docker_compose_network) \
+    -v "${PRODUCT_TESTS_ROOT}":/workspace \
+    teradatalabs/centos6-java8-oracle \
+    java -jar /workspace/build/libs/tempto-examples-all.jar \
+    --config-local /workspace/docker/tempto-configuration-docker-local.yaml \
+    $*
+}
+
+SCRIPT_DIR=$(dirname $(absolutepath "$0"))
+PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
+DOCKER_COMPOSE_LOCATION="${PRODUCT_TESTS_ROOT}/docker/docker-compose.yml"
+
+# check docker and docker compose installation
+docker-compose version
+docker version
+
+# try to stop already running containers
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" stop || true
+
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" pull
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" build
+
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d 
+docker-compose -f "${DOCKER_COMPOSE_LOCATION}" logs --no-color &
+
+docker build -t teradatalabs/presto-cli "${PRODUCT_TESTS_ROOT}"/docker/presto-cli
+retry check_hive
+retry check_presto
+
+# run product tests
+set +e
+run_product_tests "$*"
```

Comment:
This will have problem with handling `INT` or `TERM` signals. You should make a trap.